.github/workflows/xunit-tests.yml

```diff
@@ -24,8 +24,8 @@ jobs:
             z3: z3-4.8.5-x64-win
             chmod: false
             coverage: false
-          - os: macos-10.15
-            suffix: osx-10.14.1
+          - os: macos-11
+            suffix: osx-11
             z3: z3-4.8.5-x64-osx-10.14.2
```

Comment:
This change should work, but I note that publish_release_rusable.yml uses macOS-latest. Should we do that here?

Also note that there is no build of Z3 4.8.5 for a more recent macOS. I hope it does not stop working with the deprecation of 10.14.

Reply:
Yeah, we could go to `macos-latest`, which is currently `macos-11`. I'll do that.

I've generally had good luck running old binaries on newer versions of macOS, so I'm not too worried at the moment. And that's the same Z3 binary I'm currently using on my macOS 12 laptop.